### PR TITLE
Made it so user profile dropdown appears on top of mobiile content dropdown

### LINKED ISSUE #155: Year selection dropdown open when mobile hamburger also open.
Strange behavior on mobile when the user opens the hamburger menu and then opens the year selection dropdown.

### PR DESCRIPTION
**Pull request by Naapperas**
Closes #155

### CODE REVIEW

**Review by ttoino — APPROVED** [review]

File: resources/js/Components/HamburgerMenu.vue [hunk 1]
```diff
@@ -67,7 +67,7 @@ onUnmounted(() => {
     </div>
     <div
         v-show="open"
-        class="absolute left-0 top-[5.6rem] z-50 m-0 flex h-screen w-full flex-col bg-2023-teal-dark py-6 text-2xl font-semibold text-2023-bg md:hidden"
```
Comment (toni-santos) [inline resources/js/Components/HamburgerMenu.vue:70]: The `z-50` was added so that the it stays on top of the map on the home page. Removing it might break this behaviour, but I'm not sure.

**Review by toni-santos — COMMENTED** [review]
Other than the mentioned 👍

File: resources/js/Components/HamburgerMenu.vue (hunk 1 above)
Comment (Naapperas) [reply resources/js/Components/HamburgerMenu.vue:70]: Oh, I didn't actually test that, nice catch. I'll look into it.

Comment (Naapperas) [reply resources/js/Components/HamburgerMenu.vue:70]: I tested this today and it seemed fine. I might have missed some edge case but for now I'll mark this as resolved.
